horace_core/sqw/PixelData/@PixelDataBase/finalize_alignment.m

```diff
@@ -11,7 +11,7 @@
     outfile = '';
 end
 targ_obj = copy(obj);
-if ~targ_obj.is_misaligned
+if ~targ_obj.is_misaligned && targ_obj.is_range_valid
```

Comment:
This seems to be a repeated condition with a defined meaning. Possibly want a getter for this combination with a clear name.

Reply:
they appear in various independent places. Its not so big and complex thing, to define new API for it. The code is much clearer this way.